### DISCUSSION TITLE
Expected response code "250" but got code "550", with message "550 " envelope sender not allowed xyz@yahoo.com

### DISCUSSION THREAD

**Discussion by RobertIacoveciuc** [symfony/symfony#49176] [Q&A]
### Symfony version(s) affected

5.4.11

### Description

Hello, I was wondering why when i set 

` ->from($email)`

as 'xyz@yahoo.com' it is being blocked, but when I use any other email it works just fine.

My DNS config is as follows

MAILER_DSN=smtp://EMAIL_TO:PASS@OUT_DOMAIN:465

Here is the full Message

```json
{"message":"Email transport \"Symfony\\Component\\Mailer\\Transport\\Smtp\\SmtpTransport\" started","context":{},"level":100,"level_name":"DEBUG","channel":"mailer","datetime":"2023-01-31T15:07:34.034673+01:00","extra":{}}

{"message":"Send Mail Fail. \\r\\nError Code: 550 \\r\\nMessage: Expected response code \"250\" but got code \"550\", with message \"550 \"envelope sender not allowed xyz@yahoo.com\"\".","context":{},"level":600,"level_name":"EMERGENCY","channel":"app","datetime":"2023-01-31T15:07:34.046563+01:00","extra":{}}
```

### How to reproduce

Sending any email that is to @yhaoo.com

**Comment by juani0824** [comment]
I have the same problem, can you tell me how you solved it?

**Comment by lyrixx** [comment]
It looks like you are not allowed to send email from this address. 

Do you use a real mail provider (like mailgun, etc)?
Do you configure all security (like SPF, etc)?

**Comment by AndrzejW2016** [comment]
I also have this problem. Have you found a solution? I send emails to myself on behalf of my clients (history of emails sent to clients from PHP)

**Reply by dampteytsu** [reply]
Make sure the sender email has the domain name. Eg If your domain name is example.com, sender email can be 
no-reply@example.com

**Reply by sana-hany** [reply]
Thnaks very helpful
